kolibri/plugins/learn/assets/src/vue/content-card/index.vue

```diff
@@ -2,10 +2,10 @@
 
   <a v-link="link">
     <content-icon class="content-icon" v-if="kind" :kind="kind" :progress="progress"></content-icon>
-    <img :src="validatedThumbnail" class="thumbnail" v-if="showThumbnail">
+    <img :src="validatedThumbnail" class="thumbnail" v-if="showThumbnail" alt="">
     <div class="thumbnail" v-else>&nbsp;</div>
     <h3>
-      {{ title }}
+      <span class="visuallyhidden">{{ progress }} {{ kind }} </span>{{ title }}
```

Comment:
I like what you're trying to do here, but I think this may be hard to maintain when we start adding i18n to the mix. I think perhaps a better strategy would be to update the `content-icon` component (referenced above) to be more accessible by embedding a visually-hidden, screen-readable representation.